src/lib.rs

```diff
@@ -253,6 +253,10 @@ where
         self.idx += 1;
         self.packedvec.get(self.idx - 1)
     }
+
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        (0, Some(self.packedvec.len()))
```

Comment:
Would it be incorrect to set the lower bound to `self.packedvec.len()` as well?

Reply:
Good point. I should also take into account `self.idx` in the upper bound I think?

Reply:
I believe so. I think you'd return `(self.packedvec.len() - self.idx, Some(self.packedvec.len() - self.idx))`, right?